src/Common/src/Interop/Unix/libssl/SslConnectionInfo.cs

```diff
@@ -2,50 +2,34 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Security.Authentication;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.Net
 {
     internal class SslConnectionInfo
     {
-        //Windows constants for algorithms without matching enums in SslStream
-        //Taken from wincrypt.h 
-        internal const int SSL_SHA256 = 32780;
-        internal const int SSL_SHA384 = 32781;
-        internal const int SSL_ECDH = 43525;
-        internal const int SSL_ECDSA = 41475;
-
-        //Algorithm constants which are not present in wincrypt.h(not supported in Windows)
-        //These constants values are not conflicting with existing constants in wincrypt.h
-        internal const int SSL_IDEA = 229380;
-        internal const int SSL_CAMELLIA128 = 229381;
-        internal const int SSL_CAMELLIA256 = 229382;
-        internal const int SSL_eGOST2814789CNT = 229383;
-        internal const int SSL_SEED = 229384;
-
-        internal const int SSL_kPSK = 229390;
-        internal const int SSL_kGOST = 229391;
-        internal const int SSL_kSRP = 229392;
-        internal const int SSL_kKRB5 = 229393;
-
-        internal const int SSL_GOST94 = 229410;
-        internal const int SSL_GOST89 = 229411;
-        internal const int SSL_AEAD = 229412;
-
-        public readonly int Protocol;
+        public readonly SslProtocols Protocol;
```

Comment:
Note that this will cause a subtle diff. in the Windows and Unix definitions for SslConnectionInfo